db/migrate/20201006085432_create_redhat_access_telemetry_configuration.rb

```diff
@@ -0,0 +1,20 @@
+class CreateRedhatAccessTelemetryConfiguration < ActiveRecord::Migration[6.0]
```

Comment:
Older foreman versions do not support rails 6.0.
```suggestion
class CreateRedhatAccessTelemetryConfiguration < ActiveRecord::Migration[5.0]
```